packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -78,6 +78,10 @@ export class BabylonViewer extends LitElement {
       this[$scene].dispose();
     }
     this[$scene] = new Scene(this[$engine]);
+    this[$scene].imageProcessingConfiguration.toneMappingEnabled = true;
+    this[$scene].imageProcessingConfiguration.toneMappingType =
+        ImageProcessingConfiguration.TONEMAPPING_ACES;
```

Comment:
Let's adjust the exposure as well to see if we can get the renders to line up better. You can find background on why this may be necessary here: https://github.com/mrdoob/three.js/pull/19621. From that discussion, 1/0.6 might be the right factor, but please play around a bit to minimize error (just pick a few example scenes; maybe the helmets).